source/sonata_tech.rst

```diff
@@ -288,7 +288,8 @@ Group column represents the HDF group where the dataset is located under /<popul
                                                                        where :math:`ca` denotes the simulated calcium concentration in
                                                                        millimolar and :math:`y` a scalar such that at :math:`ca = 2.0:\ u_\text{final} = u`. (Markram et al., 2015)
 
-    /0            ``syn_type_id``               uint32     Mandatory   Takes the value 0 for inhibitory synapses, 100 for excitatory synapses (numerical values due to historic reasons)
+    /0            ``syn_type_id``               uint32     Mandatory   Takes the value 0 for inhibitory synapses, 100 for excitatory synapses (numerical values due to historic reasons).  Deprecated, replaced by ``model_template``.
+    /0/@library   ``model_template`             uft8       Mandatory   Reference to a template or class for the mechanism of the synapse.
```

Comment:
I didn't notice this before, but both this and `@library/spine_morphology` can be described as:
```/0/model_template utf8 Mandatory Template or class of model, should be an stored in as an enumeration in the @library
```
or something.

Reply:
OK, I was trying to be consistent with what was already there.

Reply:
Yup, completely understand, I think it's worth changing over.